### ISSUE TITLE
bunch of errors where a - was used instead of an _

### ISSUE THREAD

**Issue by zer0Kerbal** [zer0Kerbal/TheGoldStandard#4]
Hey there, I found a bunch of errors where a - was used instead of an _

For example:

```
- flowMode = ALL-VESSEL 
  - This needs to be flowMode = ALL_VESSEL 
- SCANSAT-SENSOR
- PLANETARY-RESOURCE
- RESOURCE-DEFINITION
- 
```

There may be more... not 100% sure if all of these are causing issues, but some of them are for sure.
 I see how it could have happened :)

There are a few more, you will have to go through each .cfg.

Found these in the smelters,

```
- INPUT-RESOURCE
- OUTPUT-RESOURCE
- STAGE-PRIORITY-FLOW
- 
```
by @TheKurgan

**Comment by zer0Kerbal** [comment]
apologies - a failed replace all during localization... thank you will have an update in a couple of hours.

the cause is the solution - just better regex in the global search/replace. 🧷  not find "_" replace with "-"

**Comment by zer0Kerbal** [comment]
should be fixed in #1.6.1.0

**Comment by TheKurgan** [comment]
Cheers
